1e/manifest.json

```diff
@@ -0,0 +1,60 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "d86565ae-ca60-45c2-a0b9-944ec1e05a15",
+  "app_id": "one-e",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send your 1E product metrics to Datadog.",
+    "title": "1E Integration",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "1E Digital employee experience scores",
+        "image_url": "images/score_dashboard.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Device metrics reported by 1E",
+        "image_url": "images/metrics_dashboard.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Offering::Integration",
+      "Category::Metrics"
```

Comment:
We also support new tags to specify the type of data and whether they are submitted / queried:
```
"Submitted Data Type::Metrics"
"Submitted Data Type::Logs"
"Submitted Data Type::Traces"
"Submitted Data Type::Events"
"Submitted Data Type::Incidents"
"Queried Data Type::Metrics"
"Queried Data Type::Logs"
"Queried Data Type::Traces"
"Queried Data Type::Events"
"Queried Data Type::Incidents"
```
Could you please add any relevant submitted/queried data tags?

Reply:
Added "Submitted Data Type::Metrics"